src/dist/replication/lib/replica_stub.cpp

```diff
@@ -322,6 +329,12 @@ void replica_stub::install_perf_counters()
                                                       "recent.write.busy.count",
                                                       COUNTER_TYPE_VOLATILE_NUMBER,
                                                       "write busy count in the recent period");
+
+    _counter_recent_write_size_exceed_threshold_count.init_app_counter(
+        "eon.replica_stub",
+        "recent.write.size.exceed.threshold.count",
```

Comment:
`.` is a special character in regular expression, we'd better not use it, use `_` instead.
Some old perf_counter names contain `.`, but it's not a good naming.